### PR TITLE
DOCS-3930 Moving Around the Synthetic Testing Tunnel

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Updates CI/CD Integration links, shuffles some content around, and adds aliases.

### Motivation

DOCS-3930

### Additional Notes

Remember to update the `pull_config.yaml` file before merging!

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_testing/_index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Leverage your Synthetic tests as [canary deployments][3] or run them directly in your [CI pipelines][3].
+
+{{< img src="synthetics/ci.png" alt="CI tests" style="width:100%;">}}
+
+## Examine test results in your CI environment
+
+Explore browser tests running in a CI pipeline and troubleshoot failing test results by looking at executed jobs in the [CI Results Explorer][4].
+
+{{< img src="synthetics/ci_results_explorer/ci_results_explorer_1.png" alt="CI Results Explorer" style="width:100%;">}}
```
Comment (peculiarism) [inline content/en/continuous_testing/_index.md:29]: Could we update this image with this: https://a.cl.ly/Kou1lnBQ

File: content/en/continuous_testing/_index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  text: 'Synthetics and CI/CD Troubleshooting'
+---
+
+With **end-to-end testing in production and CI environments**, your development teams can proactively ensure that defective code does not impact your users' experience. 
+
+You can run Synthetic tests on a CI pipeline in the [Datadog application][1], with the [API][2], or with [Terraform][2].
+
+## Run tests with your integration and deployment processes
+
+Leverage your Synthetic tests as [canary deployments][3] or run them directly in your [CI pipelines][3].
```
Comment (peculiarism) [inline content/en/continuous_testing/_index.md:21]:
```suggestion
Leverage your Synthetic tests as [integration and end-to-end tests][3] for your staging, pre-prod or canary deployments, or run them directly in your [CI pipelines][3].
```

File: content/en/continuous_testing/_index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+further_reading:
+- link: 'https://learn.datadoghq.com/courses/synthetic-tests-ci-cd-pipeline'
+  tag: 'Learning Center'
+  text: 'Synthetic Tests in a CI/CD Pipeline'
+- link: '/continuous_testing/troubleshooting/'
+  tag: 'Documentation'
+  text: 'Synthetics and CI/CD Troubleshooting'
+---
+
+With **end-to-end testing in production and CI environments**, your development teams can proactively ensure that defective code does not impact your users' experience.
```
Comment (peculiarism) [inline content/en/continuous_testing/_index.md:15]:
```suggestion
With **integration and end-to-end testing in production and CI environments**, your development teams can proactively ensure that defective code does not impact your users' experience. 
```

File: content/en/continuous_testing/_index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Run tests with your integration and deployment processes
+
+Leverage your Synthetic tests as [canary deployments][3] or run them directly in your [CI pipelines][3].
+
+{{< img src="synthetics/ci.png" alt="CI tests" style="width:100%;">}}
+
+## Examine test results in your CI environment
+
+Explore browser tests running in a CI pipeline and troubleshoot failing test results by looking at executed jobs in the [CI Results Explorer][4].
```
Comment (peculiarism) [inline content/en/continuous_testing/_index.md:27]:
```suggestion
Explore integration and end-to-end tests running in a CI/CD pipeline and troubleshoot failing test results by looking at executed jobs in the [CI Results Explorer][4].
```

File: content/en/continuous_testing/cicd_integrations/testing_tunnel.md [hunk 5]
```diff
@@ -21,21 +23,23 @@ further_reading:
 
 ---
 
+## Overview
+
 The Synthetic testing tunnel creates short lived secure connections between your internal environments and the Datadog infrastructure, allowing you to swiftly trigger Synthetic tests on your private applications.  
 
-Datadog recommends using the testing tunnel if you are willing to launch Synthetics tests from your CI/CD pipeline or against local versions of your application without deploying a dedicated and long lasting probing system (such as [private locations][1]). The testing tunnel can also prove helpful if you are looking at triggering tests on ephemeral cloud environments.
+Datadog recommends using the testing tunnel if you are willing to launch Synthetics tests from your CI/CD pipeline or against local versions of your application without deploying a dedicated and long lasting probing system (such as [private locations][1]). The testing tunnel is helpful if you want to trigger tests on ephemeral cloud environments.
```
Comment (peculiarism) [inline content/en/continuous_testing/cicd_integrations/testing_tunnel.md:30]:
```suggestion
Datadog recommends using the testing tunnel if you need to launch Synthetics tests against local versions of your application without deploying a dedicated and long lasting probing system (such as [private locations][1]). The testing tunnel can be used to trigger tests on ephemeral cloud environments.
```

File: content/en/continuous_testing/troubleshooting.md [hunk 6]
```diff
@@ -1,7 +1,9 @@
 ---
-title: CI/CD Integrations Troubleshooting
+title: Synthetics and CI/CD Troubleshooting
```
Comment (peculiarism) [inline content/en/continuous_testing/troubleshooting.md:2]:
```suggestion
title: Continuous Testing and CI/CD Troubleshooting
```

File: content/en/continuous_testing/troubleshooting.md [hunk 7]
```diff
@@ -1,7 +1,9 @@
...
 kind: documentation
-description: Learn about Synthetics & CI/CD concepts and troubleshoot common errors.
+description: Learn about Synthetics and CI/CD concepts and troubleshoot common errors.
```
Comment (peculiarism) [inline content/en/continuous_testing/troubleshooting.md:4]:
```suggestion
description: Learn about Continuous Testing and CI/CD concepts and troubleshoot common errors.
```

File: content/en/continuous_testing/troubleshooting.md [hunk 8]
```diff
@@ -26,11 +28,18 @@ Batch timeout
 Execution rule
 : An [execution rule][4] defines the impact of a test failure on a CI pipeline from most to least impactful: `skipped`, `non_blocking`, and `blocking`. These options are weighted and defaults to the most impactful. If your test is configured as `skipped` in the UI and `blocking` in the configuration file, it is skipped during the test run. </br><br> You can set the execution rule in your tests' properties, global configuration file, or an individual test's override file. 
 
+## CI Results Explorer
+
+### CI metadata does not appear
+
+Check whether you are using API endpoints to trigger your CI/CD test runs. In order to populate the CI Results Explorer with CI metadata, you must use the [NPM package][5].
```
Comment (peculiarism) [inline content/en/continuous_testing/troubleshooting.md:35]:
```suggestion
Check whether you are using API endpoints to trigger your CI/CD test runs. In order to populate the CI Results Explorer with CI metadata, you must use one of our integrations under [CI/CD integrations][3].
```

Thinking we should point to the GitHub Action, Circle CI orb, Azure DevOps Extension and Jenkins integration, before deferring them to the NPM package.

File: content/en/synthetics/private_locations/_index.md [hunk 9]
```diff
@@ -675,7 +675,9 @@ Because you can run several containers for one single private location with a si
 
 You can also **vertically scale** your private locations by increasing the load your private location containers can handle. Similarly, you should use the `concurrency` parameter to adjust the maximum number of test your workers allowed to run and update the resources allocated to your workers.
 
-For more information, see [Dimensioning Private Locations][18].
+For more information, see [Dimensioning Private Locations][18]. 
+
+In order to use private locations for continuous testing, the value you set for the `concurrency` parameter is the concurrency limit. For more information, see [Continuous Testing][23].
```
Comment (peculiarism) [inline content/en/synthetics/private_locations/_index.md:680]:
```suggestion
In order to use private locations for continuous testing, the value set in the `concurrency` parameter will control your parallelization. For more information, see [Continuous Testing][23].
```

I will follow up with you on this outside this PR

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 10]
```diff
@@ -346,12 +350,14 @@
       globs:
       - 'README.md'
       options:
-        dest_path: '/synthetics/cicd_integrations/'
+        dest_path: '/continuous_testing/cicd_integrations/'
         file_name: 'circleci_orb.md'
         front_matters:
           dependencies: ["https://github.com/DataDog/synthetics-ci-orb/blob/main/README.md"]
-          title: Synthetics CI CircleCI Orb
+          title: Synthetics and CI CircleCI Orb
```
Comment (peculiarism) [inline local/bin/py/build/configurations/pull_config_preview.yaml:357]:
```suggestion
          title: Continuous Testing and CircleCI Orb
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 11]
```diff
@@ -332,12 +334,14 @@
       globs:
       - 'README.md'
       options:
-        dest_path: '/synthetics/cicd_integrations/'
+        dest_path: '/continuous_testing/cicd_integrations/'
         file_name: 'github_actions.md'
         front_matters:
           dependencies: ["https://github.com/DataDog/synthetics-ci-github-action/blob/main/README.md"]
           title: Synthetics and CI GitHub Actions
```
Comment (peculiarism) [inline local/bin/py/build/configurations/pull_config_preview.yaml:341]:
```suggestion
          title: Continuous Testing and GitHub Actions
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 12]
```diff
@@ -317,13 +317,15 @@
       globs:
       - 'README.md'
       options:
-        dest_path: '/synthetics/cicd_integrations/'
+        dest_path: '/continuous_testing/cicd_integrations/'
         file_name: 'azure_devops_extension.md'
         front_matters:
           dependencies: ["https://github.com/DataDog/datadog-ci-azure-devops/blob/main/README.md"]
           title: Synthetics and Datadog CI Azure DevOps Extension
```
Comment (peculiarism) [inline local/bin/py/build/configurations/pull_config_preview.yaml:324]:
```suggestion
          title: Continuous Testing and Datadog CI Azure DevOps Extension
```

File: data/partials/home.yaml [hunk 13]
```diff
@@ -68,6 +68,10 @@ nav_sections:
         link: synthetics/
         icon: synthetics
         desc: Ensure uptime, flag regional issues, and test application performance
+      - title: Continuous Testing
+        link: continuous_testing/
+        icon: synthetics
+        desc: Implement codeless end-to-end testing using CI/CD providers
```
Comment (peculiarism) [inline data/partials/home.yaml:74]:
```suggestion
        desc: Implement codeless integration and end-to-end testing using CI/CD providers 
```

File: content/en/continuous_testing/cicd_integrations/configuration.md [hunk 14]
```diff
@@ -1,20 +1,23 @@
...
+title: Synthetics and CI/CD Configuration
 kind: documentation
-description: Configure Synthetics to run test in your CI/CD pipelines
+description: Configure Synthetics to run tests in your CI/CD pipelines.
+aliases:
+  - /synthetics/cicd_integrations/configuration
 further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-github-action-synthetics-ci-visibility/"
+  tag: "Blog"
+  text: "Use Datadog's GitHub Action to add Synthetic testing to workflows"
```
Comment (peculiarism) [inline content/en/continuous_testing/cicd_integrations/configuration.md:10]:
```suggestion
  text: "Use Datadog's GitHub Action to add continuous testing to workflows"
```

File: content/en/continuous_testing/cicd_integrations/configuration.md [hunk 15]
```diff
@@ -1,20 +1,23 @@
 ---
-title: CI/CD Integrations Configuration
+title: Synthetics and CI/CD Configuration
 kind: documentation
-description: Configure Synthetics to run test in your CI/CD pipelines
+description: Configure Synthetics to run tests in your CI/CD pipelines.
```
Comment (peculiarism) [inline content/en/continuous_testing/cicd_integrations/configuration.md:4]:
```suggestion
description: Configure Continuous Testing to run tests in your CI/CD pipelines.
```

File: content/en/continuous_testing/cicd_integrations/configuration.md [hunk 16]
```diff
@@ -1,20 +1,23 @@
 ---
-title: CI/CD Integrations Configuration
+title: Synthetics and CI/CD Configuration
```
Comment (peculiarism) [inline content/en/continuous_testing/cicd_integrations/configuration.md:2]:
```suggestion
title: Continuous Testing and CI/CD Configuration
```

File: content/en/continuous_testing/_index.md [hunk 17]
```diff
@@ -0,0 +1,38 @@
+---
+title: Continuous Testing
+kind: documentation
+description: With test automation, use fast, codeless, and reliable testing in your CI pipelines to ship features with confidence.
+disable_sidebar: true
+further_reading:
+- link: 'https://learn.datadoghq.com/courses/synthetic-tests-ci-cd-pipeline'
+  tag: 'Learning Center'
+  text: 'Synthetic Tests in a CI/CD Pipeline'
```
Comment (peculiarism) [inline content/en/continuous_testing/_index.md:9]:
```suggestion
  text: 'Continuous Testing in a CI/CD Pipeline'
```

File: content/en/continuous_testing/_index.md [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+- link: '/continuous_testing/troubleshooting/'
+  tag: 'Documentation'
+  text: 'Synthetics and CI/CD Troubleshooting'
```
Comment (peculiarism) [inline content/en/continuous_testing/_index.md:12]:
```suggestion
  text: 'Continuous Testing and CI/CD Troubleshooting'
```

File: content/en/continuous_testing/_index.md [hunk 19]
```diff
@@ -0,0 +1,38 @@
+---
+title: Continuous Testing
+kind: documentation
+description: With test automation, use fast, codeless, and reliable testing in your CI pipelines to ship features with confidence.
```
Comment (peculiarism) [inline content/en/continuous_testing/_index.md:4]:
```suggestion
description: With test automation, use fast, codeless, and reliable testing in your CI/CD pipelines to ship features with confidence.
```

File: config/_default/menus/menus.en.yaml [hunk 20]
```diff
@@ -2703,124 +2703,130 @@ main:
...
+    url: /continuous_testing/cicd_integrations/jenkins
+    parent: continuous_testing_cicd_integrations
+    identifier: continuous_testing_cicd_integrations_jenkins
+    weight: 106
+  - name: Testing Tunnel
+    url: /continuous_testing/cicd_integrations/testing_tunnel
+    parent: continuous_testing_cicd_integrations
+    identifier: continuous_testing_cicd_integraions_testing_tunnel
+    weight: 107
+  - name: CI Results Explorer
```
Comment (peculiarism) [inline config/_default/menus/menus.en.yaml:2804]:
```suggestion
  - name: Explorer
```
We're planning to rename the left nav to `Explorer` soon

**Review by peculiarism — CHANGES_REQUESTED** [review]
@alai97 A few changes, but I think we should definitely sync on this

File: content/en/continuous_testing/troubleshooting.md [hunk 21]
```diff
@@ -26,11 +28,18 @@ Batch timeout
 Execution rule
 : An [execution rule][4] defines the impact of a test failure on a CI pipeline from most to least impactful: `skipped`, `non_blocking`, and `blocking`. These options are weighted and defaults to the most impactful. If your test is configured as `skipped` in the UI and `blocking` in the configuration file, it is skipped during the test run. </br><br> You can set the execution rule in your tests' properties, global configuration file, or an individual test's override file. 
 
+## CI Results Explorer
```
Comment (alai97) [inline content/en/continuous_testing/troubleshooting.md:31]:
```suggestion
## Explorer
```

File: content/en/continuous_testing/_index.md [hunk 22]
```diff
@@ -0,0 +1,66 @@
...
+
+## Increase efficiency through seamless integrations
+
+Fast-track your application development by testing and troubleshooting in one platform. Use integrations with CI providers such as GitHub, GitLab, Jenkins, CircleCI, and Azure DevOps, and collaboration tools such as Slack and Jira to merge workflows and prevent context-switching. 
+
+You can use the [Datadog Terraform provider][4] to control test creation and state management. Leverage your Synthetic tests as [integration and end-to-end tests][5] for your staging, pre-prod, and canary deployments, or run them directly in your [CI pipelines][5].
+
+## Accelerate troubleshooting
+
+Performing tests in a unified monitoring platform allows you to quickly find the root cause of failed test runs and reduce MTTR. You can obtain the full context for troubleshooting—without switching between tools— through correlated metrics, traces, and logs surfaced by the Datadog APM integration.
```
Comment (alai97) [inline content/en/continuous_testing/_index.md:46]:
```suggestion
Performing tests in a unified monitoring platform allows you to quickly find the root cause of failed test runs and reduce MTTR. You can obtain the full context for troubleshooting—without switching between tools— through correlated metrics, traces, and logs surfaced by the Datadog [APM integration][8]. 
```

File: content/en/continuous_testing/_index.md [hunk 23]
```diff
@@ -0,0 +1,66 @@
...
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: /continuous_testing/cicd_integrations/
+[2]: /synthetics/browser_tests
+[3]: /synthetics/
+[4]: https://registry.terraform.io/providers/DataDog/datadog/latest/
+[5]: /continuous_testing/explorer
+[6]: https://app.datadoghq.com/synthetics/create#
+[7]: /continuous_testing/cicd_integrations/configuration
```
Comment (alai97) [inline content/en/continuous_testing/_index.md:66]:
```suggestion
[7]: /continuous_testing/cicd_integrations/configuration
[8]: /synthetics/apm/
```

File: content/en/continuous_testing/settings/_index.md [hunk 24]
```diff
@@ -0,0 +1,64 @@
...
+
+By default, you can run one test sequentially. To change this behavior, set a [parallelization value](#parallelization) and save your selection.
+
+## Parallelization
+
+Parallel tests are tests that run simultaneously in your continuous integration and continuous delivery (CI/CD) pipelines. 
+
+This ensures you can:
+
+* Reduce pipeline duration and ship new features faster
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/continuous_testing/settings/_index.md:36]: No, I like the variety!

File: config/_default/menus/menus.en.yaml [hunk 25]
```diff
@@ -2724,124 +2724,135 @@ main:
...
+    weight: 11
   - name: Troubleshooting
     url: synthetics/troubleshooting/
     parent: synthetics
     identifier: synthetics_troubleshooting
-    weight: 14
+    weight: 12
+  - name: Continuous Testing
+    url: continuous_testing/
+    pre: synthetics
```
Comment (alai97) [inline config/_default/menus/menus.en.yaml:2781]:
```suggestion
    pre: continuous-testing
```

**Review by peculiarism — APPROVED** [review]
giphy (2)

**Review by jtappa — CHANGES_REQUESTED** [review]
I left some suggestions, but overall the Continuous Testing index page needs some editing to make it less marketing-focused and more documentation-focused. We shouldn't be using buzzwords and jargon here, and should avoid making abstract claims about speed, reliability, or ease of use.

File: content/en/continuous_testing/_index.md [hunk 26]
```diff
@@ -0,0 +1,70 @@
+---
+title: Continuous Testing
+kind: documentation
+description: With Continuous Testing, use fast, codeless, and reliable testing in your CI/CD pipelines to ship features with confidence.
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/continuous_testing/_index.md:4]:
```suggestion
description: Set parallelization settings for Synthetic tests running in CI/CD pipelines.
```

File: content/en/continuous_testing/troubleshooting.md [hunk 27]
```diff
@@ -0,0 +1,46 @@
...
+  - /synthetics/cicd_integrations/troubleshooting
+further_reading:
+- link: "/synthetics/cicd_integrations/configuration"
+  tag: "Documentation"
+  text: "Learn how to configure Continuous Testing and CI/CD"
+---
+
+## Overview
+
+This page provides information to help you troubleshoot issues with Continuous Testing and CI/CD. If you need additional help, contact [Datadog support][1].
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/continuous_testing/troubleshooting.md:15]: Asking Beth!

File: content/en/continuous_integration/_index.md [hunk 28]
```diff
@@ -24,7 +24,7 @@ further_reading:
 <div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-<div class="alert alert-info">This page is about bringing your continuous integration (CI) metrics and data into Datadog dashboards. If you want to run Synthetic tests in your CI pipelines, see the <a href="/synthetics/cicd_integrations/" target="_blank">Synthetics and CI/CD</a> section.</div>
+<div class="alert alert-info">This page is about bringing your continuous integration (CI) metrics and data into Datadog dashboards. If you want to run Continuous Testing tests in your CI pipelines, see the <a href="/continuous_testing/cicd_integrations/" target="_blank">Continuous Testing and CI/CD</a> section.</div>
```
Comment (alai97) [inline content/en/continuous_integration/_index.md:27]:
```suggestion
<div class="alert alert-info">This page is about bringing your continuous integration (CI) metrics and data into Datadog dashboards. If you want to run Synthetic tests in your CI pipelines, see the <a href="/continuous_testing/cicd_integrations/" target="_blank">Continuous Testing and CI/CD</a> section.</div>
```

File: content/en/continuous_testing/cicd_integrations/_index.md [hunk 29]
```diff
@@ -1,10 +1,11 @@
 ---
-title: Synthetics and CI/CD
+title: Continuous Testing and CI/CD
 kind: documentation
 description: Run Synthetic tests on-demand or at predefined intervals in your CI/CD pipelines.
```
Comment (alai97) [inline content/en/continuous_testing/cicd_integrations/_index.md:4]:
```suggestion
description: Run Continuous Testing tests on-demand or at predefined intervals in your CI/CD pipelines.
```

**Pull request merged**